docs/design/manipulation/analysis.md

```diff
@@ -124,6 +124,9 @@ To implement these set of actions, the O3DE simulator should have at least:
 
  * Provide data from the simulated sensors: not in the scope of this document
 
+For an initial implementation, a lighter version of the diagram can be an intermediate goal (simplified by danielemorra98):
+
+![Manipulation light](https://user-images.githubusercontent.com/48955695/201618734-d8998631-1472-4326-822e-c5ddd66db903.svg)
```

Comment:
Are you saying you'll change it in your PR?
Either way, I changed it here too so that this PR goes in with a good version a64574464f85615f270f240fa0b979269db8d725

Currently the image is referenced twice - once in analysis.md, again in roadmap.md so that the picture can be on the same page as the roadmap for reference. It's not great to have duplicates, as we might change one and forget to change the other (I changed it in both places just now). Should we have it only in roadmap.md, and remove the light version in analysis.md?

Reply:
How about https://github.com/mabelzhang/o3de-ros2-gem/pull/1/files so we include the file in the repo and reference it from here?

> Should we have it only in roadmap.md, and remove the light version in analysis.md?

I think is fine to leave it two places, not a big deal.

Reply:
1d84eff